deployment/.env.example

```diff
@@ -0,0 +1,13 @@
+# database container
+POSTGRES_HOST="localhost"
```

Comment:
1. Yes, you're right.
2. I guess it's better to leave as it is. We also have `POSTGRES_POST` in `docker-compose.yaml`, where `POSTGRES_POST` is exposed port for postgres
3. It will be useful
4. Okay)